aspnetcore/tutorials/razor-pages/new-field.md

```diff
@@ -42,10 +42,11 @@ Automatic verification that the schema and model are in sync makes it easier to
 
    [!code-cshtml[](razor-pages-start/sample/RazorPagesMovie50/SnapShots/IndexRating.cshtml?highlight=40-42,62-64)]
 
-1. Update the following pages:
-   1. Add the `Rating` field to the Delete and Details pages.
-   1. Update [Create.cshtml](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50/Pages/Movies/Create.cshtml) with a `Rating` field.
-   1. Add the `Rating` field to the Edit Page.
+1. Update the following pages with a `Rating` field:
+   * *[Pages/Movies/Create.cshtml](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie50/Pages/Movies/Create.cshtml)*.
```

Comment:
@Rick-Anderson, do you know if we can use relative link instead of absolute link? E.g. `razor-pages-start/sample/RazorPagesMovie50/Pages/Movies/Create.cshtml`.

Reply:
We use absolute links for source (non ***.MD*** files) in GitHub